packages/core/src/database/index.ts

```diff
@@ -106,30 +106,34 @@ type PGliteDriver = {
 type PostgresDriver = {
   internal: Pool;
   user: Pool;
-  readonly: Pool;
   sync: Pool;
+  readonly: Pool;
 };
 
 type QueryBuilder = {
   /** For updating metadata and handling reorgs */
   internal: HeadlessKysely<PonderInternalSchema>;
   /** For indexing-store methods in user code */
   user: HeadlessKysely<any>;
-  /** Used in api functions */
-  readonly: HeadlessKysely<unknown>;
   /** Used to interact with the sync-store */
   sync: HeadlessKysely<PonderSyncSchema>;
+  /** Used in api functions */
+  readonly: HeadlessKysely<unknown>;
+  /** Used in client queries */
+  // client: HeadlessKysely<unknown>;
+  drizzle: Drizzle<Schema>;
+  drizzleClient: Drizzle<Schema>;
```

Comment:
Would find this clearer as `drizzleReadonly`, "client" makes it seem like it's a database client/connection object (vs a pool).